activities/community-calls/running-community-call.md

```diff
@@ -11,7 +13,7 @@ Call-in link: <https://meet.google.com/tfd-acmn-tkb>
 
 Agenda link: <https://hackmd.io/-OegeqvoThCbAsw3c3gIjw>
 
-Have each person join the call on a separate connection aligned with our [conference call etiquette](../../communication/conference-call-etiquette.md).
+Have each person join the call on a separate connection aligned with our [conference call etiquette](../communication/conference-call-etiquette.md).
 
 ### No show
 
```

Comment:
This section should probably be rephrased now when it is not specifically for the Standard community call anymore.